packages/@okta/vuepress-site/docs/guides/email-magic-links-overview/main/index.md

```diff
@@ -0,0 +1,244 @@
+
+You must create an endpoint for your application that:
+
+1. Retrieves `otp` and `state` values from the query parameters
+2. Matches the state token with the current `state` in your user's browser session
+3. Makes any other checks you deem necessary to ensure the user is working in the same browser on the same device
+4. Requests the user enter the `otp` value manually if steps 2 or 3 fail
+5. Sends `otp` and `state` to Identity Engine for validation if steps 2 and 3 pass
+6. Redirects the user to a page that continues their authentication process
+
+If your application uses the Embedded Sign-In Widget to authenticate users, pass `otp` and `state` to it as you instantiate it on the page. The Sign-In Widget handles steps 2 to 6.
+
+```javascript
+var searchParams = new URL(window.location.href).searchParams;
+var signIn = new OktaSignIn({
+   baseUrl: '${yourOktaDomain}',
+   baseUrl: '${yourOktaDomain}',
+   clientId: '${yourClientId}',
+   redirectUri: '${yourSignInRedirectUrl}',
+   useInteractionCodeFlow: true,
+   otp: searchParams.get('otp'),
+   state: searchParams.get('state')
+});
+```
+
+If your application uses a supported Embedded SDK to authenticate users, the [Okta email integration](https://developer.okta.com/docs/guides/authenticators-okta-email/aspnet/main/) and [custom password recovery](https://developer.okta.com/docs/guides/oie-embedded-sdk-use-case-custom-pwd-recovery-mfa/aspnet/main/) guides cover how to implement these steps.
+
+### Point your app's magic links to that endpoint
+
+There are two ways to set your application's magic links to the endpoint you have created:
+
+* You can set your application's Email Verification Experience (EVE) value to the endpoint's URL.
+* You can customize each of the email templates used by the email authenticator to include the endpoint's URL.
+
+Changing the EVE provides a quick way to enable and test magic links. Customizing the email templates requires more work but results in fewer roundtrips to the Identity Engine and a faster user experience.
+
+#### Using the Email Verification Experience
+
+To enable magic links in your application using the email verification experience (EVE) setting:
+
+1. Open the **Admin Console** for your Okta org.
+2. Choose **Applications > Applications** to show the app integrations that you have already created.
+3. Click the application that you previously created.
+4. In the **General Settings** section on the **General** tab, click **Edit**.
+5. Under **EMAIL VERIFICATION EXPERIENCE** enter the URL for the endpoint you created. For example, `https://example.com/magiclink/callback`.
+6. Click **Save** to save your changes.
+
+When required, Identity Engine sends an email similar to this one containing an OTP and a link:
+
+<div class="half border">
+
+![A sample email sent by the Identity Engine with OTP and magic link highlighted](/img/advanced-use-cases/email-magic-links-overview-sample-email.png)
```

Comment:
I've deleted the border style. Whatever we end up doing with the images I don't imagine it will involve putting a border all the way aroudn the edge of this image.